language/de/acp/forums.php

```diff
@@ -90,7 +90,7 @@
 	'FORUM_DELETE_EXPLAIN'				=> 'Das folgende Formular erlaubt dir, ein Forum zu löschen. Wenn in dem Forum Beiträge erstellt werden können, kannst du entscheiden, was mit den darin enthaltenen Themen (oder Foren) geschehen soll.',
 	'FORUM_DELETED'						=> 'Forum wurde erfolgreich gelöscht.',
 	'FORUM_DESC'						=> 'Beschreibung',
-	'FORUM_DESC_EXPLAIN'				=> 'Jede verwendete HTML-Auszeichnung wird so, wie sie ist, angezeigt.',
+	'FORUM_DESC_EXPLAIN'				=> 'Jede verwendete HTML-Auszeichnung wird so, wie sie ist, angezeigt. Wenn als Forum-Typ „Kategorie“ ausgewählt ist, wird die Beschreibung nicht verwendet.',
```

Comment:
Wenn wir schon in der Zeile sind: "Jede verwendete HTML-Auszeichnung wird so angezeigt, wie sie ist." ?

Reply:
Ich finde das etwas verwirrend geschrieben. Ziel ist doch zu sagen, dass sich HTML verwenden lässt, nicht dass HTML Tags für den Benutzer sichtbar angezeigt werden. Also vielleicht schlicht 
"HTML Auszeichnungen sind zur Formatierung verwendbar."?

Reply:
Das wäre sachlich falsch.

Reply:
Wenn HTML-Codes nicht interpretiert werden, sondern dann als Text ausgegeben werden, würde ich eher schreiben:
Eine Verwendung von HTML-Auszeichnungen ist nicht möglich.